front/src/pages/configurator/[base64].tsx

```diff
@@ -1,18 +1,18 @@
-import { Box, MantineProvider } from "@mantine/core";
+import { Box } from "@mantine/core";
 import { useRouter } from "next/router";
 import { Engine } from "../../rendering-engine/components/engine";
 import { useGetBuildTreeByBase64Code } from "../../rendering-engine/service/configuratorService";
 import Link from "next/link";
-import {FRONT_CURRENT_PATH} from "../../config/env-paths";
+import { FRONT_CURRENT_PATH } from "../../config/env-paths";
 
-const Configurator = ({ testData }) => {
+const Configurator = () => {
     const router = useRouter();
     const { base64 } = router.query;
 
     const [data] = useGetBuildTreeByBase64Code(base64 as string);
 
     return (
-        <MantineProvider withGlobalStyles withNormalizeCSS>
+        <>
```

Comment:
Why we delete provider?

Reply:
Because we already have one provider on the app level. This one was a duplicate.